pkg/network/network_test.go

```diff
@@ -0,0 +1,66 @@
+package network
```

Comment:
please rename the module to kubemacpool_test.go, so we don't mix all tests together later